content/scriptprefs.js

```diff
@@ -57,11 +57,11 @@ window.addEventListener('load', function() {
 }, false);
 
 function onDialogAccept() {
-  gScript.includes = gScriptIncludesEl.pages;
+  // gScript.includes = gScriptIncludesEl.pages;
```

Comment:
Oh; of course.  You can't edit these fields so we shouldn't need to read their values.  Yes?